dbt/models/default/docs.md

```diff
@@ -1,49 +1,59 @@
-{% docs vw_pin_history %}
-Current and prior years' assessments by PIN in wide format.
+# vw_card_res_char
+
+{% docs view_vw_card_res_char %}
+View to standardize residential property characteristics for use in modeling
+and reporting.
 
 ### Nuance
 
-`spatial.township` is not yearly.
+- Only contains characteristics for "regression class" residential properties,
+  see `ccao.class_dict` for details.
+- Observations are card-level, i.e. each row is one building. Note that a
+  card does not necessarily equal a PIN.
+- Land is parcel-level, not card-level.
 {% enddocs %}
 
-{% docs vw_pin_value %}
-CCAO mailed total, CCAO final, and BOR final values for each PIN by year.
+# vw_pin_address
+
+{% docs view_vw_pin_address %}
+Source of truth view for PIN address, both legal (property address)
+and mailing (owner/taxpayer address).
 
 ### Assumptions
-Taking the max value by 14-digit pin and year is sufficient for accurate values.
-We do this because even given the criteria to de-dupe `asmt_all`, we still end
-up with duplicates by pin and year.
-{% enddocs %}
 
-{% docs vw_pin_appeal %}
-Appeals by PIN.
+- Mailing addresses are updated when properties transfer ownership.
 
-### Assumptions
+### Nuance
 
-* Appeal types coding is static
-* Status coding is static
-* Condo/coop appeal type is new, so we have to assume which classes fit the
-  category prior to 2021
+- Newer properties may be missing a mailing or property address, as they
+  need to be assigned one by the postal service.
 {% enddocs %}
 
-{% docs vw_card_res_char %}
-View that standardizes residential property characteristics for use in modeling
-and reporting.
+# vw_pin_appeal
+
+{% docs view_vw_pin_appeal %}
+View of assessment appeals by stage (wide format). Shows appeal decision,
+reason, and results.
 
 ### Assumptions
 
-### Nuance
-Land is parcel-level.
-{% enddocs %}
+- Appeal types coding is static.
+- Status coding is static.
+- Condo/co-op appeal type is new, so we have to assume which classes fit the
+  category prior to 2021.
 
-{% docs vw_pin_address %}
-Source of truth view for PIN address, both legal and mailing.
+### Nuance
 
-### Assumptions
-Mailing addresses are updated when properties transfer ownership.
+- Not live updated. This table is only updated as townships are "closed"
```

Comment:
Are we sure this is the case? I create appeal workbooks for commercial from htpar that are used _to_ close townships.

Reply:
Fixed in 268e53563c2fcca2d0e397eaad6b4c2296345daa!